docs/sankey.md

```diff
@@ -0,0 +1,109 @@
+# Sankey
+
+Note: This component is in alpha.
+
+### Usage
+
+```jsx
+import {Sankey} from 'react-vis';
+
+const nodes = [{name: 'a'}, {name: 'b'}, {name: 'c'}];
+const links = [
+  {source: 0, target: 1, value: 10},
+  {source: 0, target: 2, value: 20},
+  {source: 1, target: 2, value: 20}
+];
+
+<Sankey
+  nodes={nodes}
+  links={links}
+  width={200}
+  height={200}
+/>
+```
+
+### Api
+
+##### width (required)
+##### height (required)
+##### nodes (required)
+
+An array of objects matching the following shape:
+
+```
+{
+  color?: String,
+  opacity?: Number,
+  key?: String
+}
+```
+
+All these fields are optional.
+
+##### links (required)
+
+An array of objects matching the following shape:
+
+```
+{
+  source: Number,
+  target: Number,
+  value: Number,
+  color?: String,
+  opacity?: Number,
+  key?: String
+}
+```
+
+Where both `source` and `target` are the index of the nodes they intent to represent.
+Other fields are optional.
+
+##### margin
+
+The margin that will be applied to each side of the Sankey.
+
+Defaults to `20`.
+
+##### nodeWidth
+
+Width of the nodes.
+
+Defaults to `10`.
+
+##### nodePadding
+
+Padding between each node.
+
+Defaults to `10`.
+
+##### align
+
+The alignment used for the sankey.
+Can be `justify`, `center`, `left`, `right`.
+
+Defaults to `justify`.
+
+##### layout
+
+The number of passes the sankey algorithm will do in order to arrange positioning.
+
+Defaults to `50`.
+
+##### hasVoronoi
+
+Determine if the node selection will be done using a voronoi or not. Although less
```

Comment:
i want a diagram explaining why/how this works. generate a version of the voronoi where it has the outlines drawn in